src/_playground/documentation/utils/getSourceModules.js

```diff
@@ -0,0 +1,9 @@
+const getSourceModules = (sourceModulePath) => {
+    const cleanSource = sourceModulePath.replace('./src', '');
+
+    const sourceModules = require('../../../../src' + cleanSource);
```

Comment:
I know we trudged through getting this to work, but it might be worth dropping a comment in here as to _why_ we are stripping out `src` and then adding it back in and _why_ we're doing the string concatenation the way we are.